test/hotspot/jtreg/gc/cslocker/TestCSLocker.java

```diff
@@ -51,7 +51,6 @@ public static void main(String args[]) throws Exception {
 
         // check timeout to success deadlocking
         while(System.currentTimeMillis() < startTime + timeout) {
```

Comment:
In addition to @albertnetymk's comment, please also fix the whitespace after the `while` and the bracket.